src/main/scala/com/grandata/commons/dates/DateRange.scala

```diff
@@ -13,4 +13,11 @@ object DateRange {
 class DateRange(val start: LocalDate, val end: LocalDate) {
   def dayIterator = DateUtils.dayIterator(start, end)
   def monthIterator = DateUtils.monthIterator(start, end)
+  
+  override def hashCode = 41 * (41 + start.hashCode) + end.hashCode
+  
+  override def equals(other: Any) = other match {
+    case that: DateRange => this.start == that.start && this.end == that.end
+    case _ => false
+  }
```

Comment:
A little more nice:
override def equals(other: Any) =  this.start == that.start && this.end == that.end